docs/source/development/set_up_vscode.md

```diff
@@ -242,7 +242,7 @@ Go to the **Debugging** section in VS Code and select the newly created remote d
 
 ![](../meta/images/vscode_remote_debugger.png)
 
-You will need to set a breakpoint in VS Code as described [above](#debugging) and start the debugger by clicking the green play triangle:
+You will need to [set a breakpoint in VS Code as described in the debugging section above](#debugging) and start the debugger by clicking the green play triangle:
```

Comment:
You must [set a breakpoint in VS Code as described in the debugging section above](#debugging) and start the debugger by clicking the green play triangle: